packages/nextjs/test/serverIntegrations.test.ts

```diff
@@ -0,0 +1,54 @@
+import { RewriteFrames } from '@sentry/integrations';
+import { Integration } from '@sentry/types';
+
+import { defaultRewriteFrames, getFinalServerIntegrations } from '../src/utils/serverIntegrations';
+
+interface IntegrationFunction {
+  (integrations: Integration[]): Integration[];
+}
+
+describe('integrations without RewriteFrames', () => {
+  test('as an array', () => {
+    const integrations: Integration[] = [];
+    // Should get a single integration: RewriteFrames
+    let finalIntegrations = getFinalServerIntegrations(integrations);
+    expect(Array.isArray(finalIntegrations)).toBeTruthy();
+    finalIntegrations = finalIntegrations as Integration[];
+    expect(finalIntegrations.length === 1).toBeTruthy();
```

Comment:
I like to save `toBeTruthy` for times when I'm testing a value and truthiness is actually what I want (IOW, when I don't care what it is, as long as it's not falsey). Here I'd at minimum go for `.toBe(true)` instead (we know it's a boolean, after all, not just a non-falsey value), but even better I think would be:

```suggestion
    expect(finalIntegrations.length).toEqual(1);
```

(Similar to the logic above, I try to save `.toBe(true)` for things where I'm testing a boolean value. Here you've _created_ a boolean value by testing against a number, but the thing we're actually interested in is the number itself. Also, if you don't follow the pattern of saving `.toBe(true)` for boolean values, there's nothing to stop you from having nearly every single expect statement end with `.toBe(true)` or `.toBe(false)`, and that's a lot of extra typing energy/reading energy when you add it all up. Laziness FTW!)

Reply:
`expect(finalIntegrations).toHaveLength(1);` :)